docs/platforms/android/integrations/okhttp/index.mdx

```diff
@@ -2,25 +2,25 @@
 title: OkHttp
 caseStyle: camelCase
 supportLevel: production
-sdk: sentry.java.android.okhttp
+sdk: sentry.java.okhttp
 description: "Learn more about the Sentry OkHttp integration for the Android SDK."
 categories:
   - mobile
 ---
 
-The `sentry-android-okhttp` library provides [OkHttp](https://github.com/square/okhttp) support for Sentry via the [OkHttp Interceptor](https://github.com/square/okhttp/blob/a2059dedc0b1d4a977480834ae4ed9ea576a3eb8/okhttp/src/main/kotlin/okhttp3/Interceptor.kt). The source can be found [on GitHub](https://github.com/getsentry/sentry-java/tree/main/sentry-android-okhttp/src/main/java/io/sentry/android/okhttp).
```

Comment:
I removed our the GitHub link because we already have it in the right sidebar and also link to the interceptor implementation further below